oak_restricted_kernel_bin/Cargo.toml

```diff
@@ -27,6 +28,13 @@ test = false
 bench = false
 required-features = ["virtio_console_channel"]
 
+[[bin]]
+path = "src/main.rs"
```

Comment:
IIRC cargo complains about sources used for multiple bins -- or is that not the case any more?